### PR TITLE
Feat:Added Loading Animation

### PR DESCRIPTION
**Pull request by kiranpratyush**
Added loading Animation

https://user-images.githubusercontent.com/77445957/163659744-cfbca814-218d-4ff8-9d08-106140b1ac2b.mp4

### CODE REVIEW

File: src/Components/Skeleton/skeleton.css [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  align-items: center;
+}
+.skeleton.text {
+  width: 80%;
+  height: 30px;
+}
+.skeleton.text-small {
+  width: 50%;
+  height: 20px;
+  margin:10px 0 10px 54px;
```
Comment (KoushikKintali) [inline src/Components/Skeleton/skeleton.css:19]: use rems instead of px if > 10 px

**Review by KoushikKintali — APPROVED** [review]
skeleton animation is good!!!

**Review by manojasarsa — APPROVED** [review]
Looks good!

File: src/Components/Skeleton/skeleton.css (hunk 1 above)
Comment (kiranpratyush) [reply src/Components/Skeleton/skeleton.css:19]: yes Thank you

**Pull request merged**